src/pages/trails/ballots/components/ballot-form.vue

```diff
@@ -132,12 +145,17 @@ export default {
     file: function () {
       this.convertToIFPS(this.file)
     },
-    account: function (account) {
+    account: async function (account) {
       this.fetchTreasuriesForUser(account)
+      const getAccount = await this.$store.$api.getAccount(this.account)
+      this.userBalance = getAccount.core_liquid_balance.replace(/[^0-9]/gi, '')
```

Comment:
Since we're using `replace(/[^0-9]/gi, ''` in two places already, I recommend separating it into a function. I would also recommend adding a small comment explaining the regular expression and a example of what it does for future people looking into the code.